test/test_gmm.py

```diff
@@ -344,6 +344,36 @@ def test_tgmm_bf16(self):
     # Make sure tgmm doesn't fallback.
     self.assertNotIn("aten::", met.short_metrics_report())
 
+  @unittest.skipIf(xr.device_type() != 'TPU', "This test only works on TPU.")
+  def test_gmm_backward(self):
+    self._init_test_cases()
```

Comment:
nit, you might need a `met.clear_all()` here.

Reply:
Let me fix it in the next PR. Don't want to waste CI cycles.